src/gluonnlp/metric/masked_accuracy.py

```diff
@@ -0,0 +1,304 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Masked accuracy metric."""
```

Comment:
Why do we need accuracy for masked language model? Based on the original paper, only Next Sentence Prediction need the accuracy, am I right?

Reply:
It would be good if we can also provide `weights` of different prediction in the `MaskedAccuracy `.